webroot/js/directives/sentence-and-translations.dir.js

```diff
@@ -76,28 +76,38 @@
         };
     }
 
-    function SentenceAndTranslationsController($http, $cookies) {
+    function SentenceAndTranslationsController($rootScope, $scope, $http, $cookies, $timeout) {
```

Comment:
I thought that declaring the injections like that, as opposed to e.g.
```js
.controller('SentencesListsDownloadCtrl', ['$scope', '$http', '$timeout', function ($scope, $http, $timeout)`
```
wasn’t working because the JS minifier is renaming the parameters.

Reply:
Encapsulating the code inside an "Immediately Invoked Function Expression" prevents issues with minification, if I'm not mistaken.

I mean this thing:

```
(function() {

// code

})();
```

Reply:
No, this is not what I’m talking about. Your code currently works because that file isn’t minified yet. However if we minify it, it *will* break. Have a look at `config/asset_compress.ini` `src/Template/Layout/default.ctp` to see how minification is handled.

Reply:
You're right.

I pushed a commit that minifies the `sentence-and-translations.dir.js` while I was at it.

Reply:
Great! :+1: